tests/api_tests/users/test_views.py

```diff
@@ -203,6 +204,313 @@ def test_get_projects_logged_in_as_different_user(self):
         assert_not_in(self.deleted_project_user_one._id, ids)
 
 
+class TestUserUpdate(ApiTestCase):
+
+    def setUp(self):
+        super(TestUserUpdate, self).setUp()
+        self.user_one = UserFactory.build()
+        self.user_one.set_password('justapoorboy')
+        self.user_one.fullname = 'Martin Luther King Jr.'
```

Comment:
This is fine, but you can make this much more concise by passing the same kwargs to `build()` that you would pass to the `User` constructor.

```python
UserFactory.build(
    fullname='Martin Luther King Jr.',
    given_name='Martin',
    #...
)
```